sdk/eventhubs/azure_messaging_eventhubs/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 1.0.0-beta.1 (2025-02-13)
```

Comment:
Until 1.0, we are using this syntax. Use 0.1.0 or whatever would be first for your crate name. That's idiomatic to rust.